keyboards/ferris/sweep/config.h

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2018 ENDO Katsuhiro <ka2hiro@curlybracket.co.jp>
```

Comment:
Shouldn't this be something like:

```suggestion
Copyright 2020 David Philip Barr @davidphilipbarr
```

Reply:
I don't know. I took that line from DPB's repo, and thought they kept this from whatever they based their work on, so kept it as is. I guess the most correct alternative would be to append their name, and then may as well append mine too 🙂

Reply:
There's a [section in the docs](https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=copyright-blurb) that goes over this.

Correct solution here from what you've told me is that it should have both David's name and yours.

Reply:
I guess it depends on what's considered trivial or not :) I've updated the PR to add DPB and myself :)